utils/spacewalk-utils.changes

```diff
@@ -1,3 +1,4 @@
+- require python3 only
```

Comment:
Strictly speaking, this is not only about requiring, but also about building and providing, right?

Reply:
I have changed the message to the same as we have for spacewalk-proxy